pkg/ds/farm_test.go

```diff
@@ -740,6 +751,8 @@ func TestMultipleFarms(t *testing.T) {
 	}
 	_, err = dsStore.MutateDS(anotherDSData.ID, mutator)
 	Assert(t).IsNil(err, "Expected no error mutating daemon set")
+	// Wait double the time because testing multiple farms is that slow
```

Comment:
It doesn't seem right to wait an arbitrary amount of time for the farm to be available. Eventually (probably on Travis-CI) you're going to have test runs with high CPU contention and this timeout value isn't going to matter. Is there property you can poll to verify that the farm is in the state you want?